XamlControlsGallery/ControlPages/PipsPagerPage.xaml

```diff
@@ -0,0 +1,85 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.PipsPagerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AppUIBasics"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="PipsPager integrated with a FlipView">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+                        <FlipView.ItemTemplate>
+                            <DataTemplate x:DataType="x:String">
+                                <Image Source="{x:Bind Mode=OneWay}"/>
+                            </DataTemplate>
+                        </FlipView.ItemTemplate>
+                    </FlipView>
+                    <muxc:PipsPager x:Name="FlipViewPipsPager"
+                        HorizontalAlignment="Center"
+                        Margin="0, 10, 0, 0"
```

Comment:
```suggestion
                        Margin="0, 12, 0, 0"
```

This likely doesn't actually matter, but I try to keep to a 4px design grid. And at least in the past, 12px and 24 px have been the two most common margin sizes that design recommends.